pkg/queue/health/handler.go

```diff
@@ -30,37 +31,40 @@ const badProbeTemplate = "unexpected probe header value: "
 
 // ProbeHandler returns a http.HandlerFunc that responds to health checks.
 // This handler assumes the Knative Probe Header will be passed.
-func ProbeHandler(prober func() bool, tracingEnabled bool) http.HandlerFunc {
+func ProbeHandler(tracer trace.Tracer, prober func() bool) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		ph := netheader.GetKnativeProbeValue(r)
 
-		var probeSpan *trace.Span
-		if tracingEnabled {
-			_, probeSpan = trace.StartSpan(r.Context(), "probe")
-			defer probeSpan.End()
-		}
+		_, probeSpan := tracer.Start(r.Context(), "kn.queueproxy.probe")
+		defer probeSpan.End()
```

Comment:
here, we don't add the ctx from the tracer back to the request because there are no downstream spans we want to add?

Reply:
I don't know offhand, it was like this before - I created a follow up issue to revisit this - https://github.com/knative/serving/issues/15962